verilog/analysis/verilog_linter.cc

```diff
@@ -383,22 +383,48 @@ void GetLintRuleDescriptionsHelpFlag(std::ostream* os,
   }
 }
 
-void GetLintRuleDescriptionsMarkdown(std::ostream* os) {
+void GetLintRuleFile(std::ostream *os) {
+  // Set up the map.
+  auto rule_map = analysis::GetAllRuleDescriptions();
+  for (const auto &rule_id : analysis::kDefaultRuleSet) {
+    rule_map[rule_id].default_enabled = true;
+  }
+
+  // Print all the rules
+  for (const auto &rule : rule_map) {
+    *os << (rule.second.default_enabled ? "+" : "-");
+    *os << rule.second.descriptor.name;
+
+    bool first = true;
+    for (auto it = rule.second.descriptor.param.begin();
+         it != rule.second.descriptor.param.end(); ++it) {
+      *os << (first ? "=" : ";");
+      *os << it->name << ":";
+      *os << it->default_value;
```

Comment:
(Note: just something for possible future change)
Maybe something to consider in the future: if the user has set values on the command line with the `--rules` flag, the output would not reflect that as we only print the defaults here. There might be good reasons for both cases - default to get a starting files and the other after someone has optimized their settings on the command line and wants to print a file with all the settings

Reply:
I ran into the use case you mentioned so I've implemented this change.